plugins/commands/cloud/util.rb

```diff
@@ -1,205 +1,309 @@
+      # @return [nil]
+      def format_search_results(search_results, short, json, env)
+        result = search_results.map do |b|
+          {
+            name: b.tag,
+            version: b.current_version.version,
+            downloads: format_downloads(b.downloads.to_s),
+            providers: b.current_version.providers.map(&:name).join(", ")
+          }
         end
 
-        def write_line(env, columns,h,to_rjust_keys)
-          str = h.keys.map { |k|
-            if to_rjust_keys.include?(k)
-              h[k].rjust(columns[k][:width])
-            else
-              h[k].ljust(columns[k][:width])
-            end
-          }.join(" | ")
-          env.ui.info "| #{str} |"
+        if short
+          result.map { |b| env.ui.info(b[:name]) }
+        elsif json
+          env.ui.info(result.to_json)
+        else
+          column_labels = {}
+          columns = result.first.keys
+          columns.each do |c|
+            column_labels[c] = c.to_s.upcase
+          end
+          print_search_table(env, column_labels, result, [:downloads])
         end
+        nil
+      end
 
-        # ===================================================
-        # ===================================================
+      # Output box details result from Vagrant Cloud
+      #
+      # @param [VagrantCloud::Box, VagrantCloud::Box::Version] box Box or box version to display
+      # @param [Vagrant::Environment] env Current Vagrant environment
+      # @return [nil]
+      def format_box_results(box, env)
+        if box.is_a?(VagrantCloud::Box)
+          info = box_info(box)
+        elsif box.is_a?(VagrantCloud::Box::Provider)
+          info = version_info(box.version)
+        else
+          info = version_info(box)
+        end
 
-        # Takes a "mostly" flat key=>value hash from Vagrant Cloud
-        # and prints its results in a list
-        #
-        # @param [Hash] - results - A response hash from vagrant cloud
-        # @param [Vagrant::Environment] - env
-        def format_box_results(results, env)
-          # TODO: remove other description fields? Maybe leave "short"?
-          results.delete("description_html")
+        width = info.keys.map(&:size).max
+        info.each do |k, v|
+          whitespace = width - k.size
+          env.ui.info "#{k}: #{"".ljust(whitespace)} #{v}"
+        end
+        nil
+      end
 
-          if results["current_version"]
-            versions = results.delete("versions")
-            results["providers"] = results["current_version"]["providers"]
+      # Load box and yield
+      #
+      # @param [VagrantCloud::Account] account Vagrant Cloud account
+      # @param [String] org Organization name
+      # @param [String] box Box name
+      # @yieldparam [VagrantCloud::Box] box Requested Vagrant Cloud box
+      # @yieldreturn [Integer]
+      # @return [Integer]
+      def with_box(account:, org:, box:)
+        org = account.organization(name: org)
```

Comment:
running commands against orgs that don't exist produce an error here.
Eg. running the command `vagrant cloud box show sdgf/sgdf` produces

```
bundler: failed to load command: vagrant (/Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/bin/vagrant)
Excon::Error::NotFound: Expected([200, 201, 204]) <=> Actual(404 Not Found)

  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/expects.rb:13:in `response_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/response_parser.rb:12:in `response_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/connection.rb:456:in `response'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/connection.rb:287:in `request'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/idempotent.rb:50:in `error_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/base.rb:17:in `error_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/base.rb:17:in `error_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/connection.rb:312:in `rescue in request'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/connection.rb:232:in `request'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/idempotent.rb:50:in `error_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/base.rb:17:in `error_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/middlewares/base.rb:17:in `error_call'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/connection.rb:312:in `rescue in request'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/excon-0.78.0/lib/excon/connection.rb:232:in `request'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/vagrant_cloud-3.0.1/lib/vagrant_cloud/client.rb:129:in `block in request'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/vagrant_cloud-3.0.1/lib/vagrant_cloud/client.rb:84:in `block in with_connection'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/vagrant_cloud-3.0.1/lib/vagrant_cloud/client.rb:84:in `synchronize'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/vagrant_cloud-3.0.1/lib/vagrant_cloud/client.rb:84:in `with_connection'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/vagrant_cloud-3.0.1/lib/vagrant_cloud/client.rb:129:in `request'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/vagrant_cloud-3.0.1/lib/vagrant_cloud/client.rb:227:in `organization_get'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/gems/vagrant_cloud-3.0.1/lib/vagrant_cloud/account.rb:99:in `organization'
  /Users/sophia/project/vagrant/plugins/commands/cloud/util.rb:144:in `with_box'
  /Users/sophia/project/vagrant/plugins/commands/cloud/box/show.rb:56:in `show_box'
  /Users/sophia/project/vagrant/plugins/commands/cloud/box/show.rb:40:in `execute'
  /Users/sophia/project/vagrant/plugins/commands/cloud/box/root.rb:46:in `execute'
  /Users/sophia/project/vagrant/plugins/commands/cloud/root.rb:73:in `execute'
  /Users/sophia/project/vagrant/lib/vagrant/cli.rb:67:in `execute'
  /Users/sophia/project/vagrant/lib/vagrant/environment.rb:290:in `cli'
  /Users/sophia/project/vagrant/bin/vagrant:205:in `<top (required)>'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/bin/vagrant:23:in `load'
  /Users/sophia/miniconda3/envs/vagrant/lib/ruby/gems/2.6.0/bin/vagrant:23:in `<top (required)>'
```

Reply:
Fixed via hashicorp/vagrant_cloud#67